README.md

```diff
@@ -2,9 +2,69 @@
 
 ## Introduction
 
-**TiledCUDA** is a kernel template library that is designed to be highly efficient. It provides a wrapper for cutlass **CuTe** to simplifly the process of implementing complex fused kernels that utilize tensor core GEMM.
+**TiledCUDA** is a kernel template library that is designed to be highly efficient and easy to use. 
 
-TiledCUDA utilizes **PyTorch** as its runtime environment and leverages the **Tensor** class of PyTorch for convenient testing.
+TiledCUDA takes a bottom-up approach, building efficient kernel components centered around **Base Tiles**.These components encapsulate performance-related parameters that users can configure according to their specific requrements.
+
+The features of TiledCUDA include:
+
+- **High-level API**: TiledCUDA provides a high-level API that is easy to use and understand.
+- **Flexible**: TiledCUDA allows users to configure the kernel components according to their specific requirements.
+- **Efficient**: TiledCUDA provides efficient implementations of the kernel components.
+
+## Design
+
+TiledCUDA builds kernels around the core concept of **BaseTile**, starting from the lowest level by encapsulating atomic instructions(`ldmatrix`, `stmatrix`, `mma`, etc.), and then composing them step-by-step in both the temporal and spatial domains.
```

Comment:
TiledCUDA builds kernels around the core concept of **BaseTile**, which is aligned with a shape that optimally utilizes TensorCore's capabilities. Larger tiles are then constructed from BaseTile in both temporal and spatial dimensions.